.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+<!--
+
+A crisp summary
+
+Fixes #
+Ref #
+-->
+
+<!--
+Apply labels!
+-->
+
+## Checklist
+
+- [ ] Labels applied correctly? See the [contributing guidlines](../CONTRIBUTING.md).
+- [ ] Workspace compiles `cargo check --workspace --tests`?
+- [ ] Workspace tests `cargo test --workspace`?
+- [ ] Applied `cargo +nightly fmt` with the version from CI?
+- [ ] Ran `cargo spellcheck` without any fallout? If any, update `.spellcheck/lingua.dic`.
```

Comment:
All that is ensured by the CI, not sure why this needs to be mentioned here.

Reply:
I think, guidance on _how_ to please the CI is a generally good and nice thing to state in a PR. The above might indeed be too verbose, but i.e. `cargo-spellcheck` and `cargo fmt` are something that wasn't covered before, but we only point to a style guide that is not so accurate anymore.

Reply:
We can remove the checklist property and just make it a list of things to cover.

Reply:
Sure, works as well